src/EnergyPlus/IntegratedHeatPump.cc

```diff
@@ -198,18 +198,19 @@ namespace EnergyPlus {
 						SimVariableSpeedCoils( BlankString, IntegratedHeatPumps( DXCoilNum ).SCWHCoilIndex,
 						                       CyclingScheme, MaxONOFFCyclesperHour, HPTimeConstant, FanDelayTime,
 						                       CompOp, 0.0, 1, 0.0, 0.0, 0.0, OnOffAirFlowRat );
+						SimVariableSpeedCoils( BlankString, IntegratedHeatPumps( DXCoilNum ).DWHCoilIndex,
+							CyclingScheme, MaxONOFFCyclesperHour, HPTimeConstant, FanDelayTime,
+							CompOp, 0.0, 1, 0.0, 0.0, 0.0, OnOffAirFlowRat);
 
 						SimVariableSpeedCoils( BlankString, IntegratedHeatPumps( DXCoilNum ).SCCoilIndex,
-						                       CyclingScheme, MaxONOFFCyclesperHour, HPTimeConstant, FanDelayTime,
-						                       CompOp, PartLoadFrac,
-						                       SpeedNum, SpeedRatio, SensLoad, LatentLoad, OnOffAirFlowRat );
+							CyclingScheme, MaxONOFFCyclesperHour, HPTimeConstant, FanDelayTime,
```

Comment:
@shenb It appears that these changes are just indentation?  Several others like that below.